packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -114,6 +114,16 @@ The following table provides a list of all available email templates in an Okta
 | Reassigned Review | Access certification campaign: (`campaignName`) | `IGAReviewerReassigned` | | Sent to reviewers when they are assigned new reviews by an admin or a different reviewer. |
 | Idp MyAccount Email Change Confirmation | Confirm email address change | `MyAccountChangeConfirmation` |  | Sent to users who try to verify an email address using MyAccount APIs. The users must enter the provided code to confirm the change. |
 
+### Use Branding variables
+
+After you create a theme for your org, you can use branding variables in your custom email templates by selecting the `FULL_THEME` variant for your emails. If you delete any of these optional variables from the email code, it will disconnect the theme object properties from the email template.
```

Comment:
The variant is in the UI is displayed as `Solid background`

Reply:
hmm the UI is that, but the API variant name is FULL_THEME, should we refer to both? or since this is dev docs, that's why I put the API name not the UI name

Reply:
@brentschaus-okta ^ question is for you sorry

Reply:
Is there value in including both?

Reply:
`Solid Background` == `FULL_THEME` 

yes, variables in this table are included by default in all emails, is the same variant

Reply:
I just added `Solid Background` in parenthesis so both API and UI terms are there, let me know if anyone disagrees